std/file.d

```diff
@@ -982,27 +994,36 @@ void setTimes(in char[] name,
                          FILE_ATTRIBUTE_DIRECTORY |
                          FILE_FLAG_BACKUP_SEMANTICS,
                          HANDLE.init);
-        auto h = trustedCreateFileW(name, defaults);
+        auto h = trustedCreateFileW(namez, defaults);
 
-        cenforce(h != INVALID_HANDLE_VALUE, name);
+        static if (isNarrowString!R && is(Unqual!(ElementEncodingType!R) == char))
```

Comment:
I do think long term we need to do something better about this. For now this is fine because it's internal only.